cmd/resource-request-operator/main.go

```diff
@@ -0,0 +1,77 @@
+package main
```

Comment:
IIUC the controller was added to the manager of advertisement operator. You should drop this then.